solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import {
+  AccordionItemHeading,
+  AccordionItemButton,
+  AccordionItemState,
+} from "react-accessible-accordion"
+
+const AccordionItemHeadingWithPlusMinusSign = ({
+  children,
+  className = "",
+  ...accordionItemButtonProps
+}) => {
+  return (
+    <AccordionItemHeading
+      className={`accordion__heading--with-plus-minus-sign ${className}`}
```

Comment:
Based on the BEM docs we should `accordion__heading accordion__heading--with-plus-minus-sign`.

Reply:
Fixed in b5e3837efee85365e215e2b9166c0c7f59ab563c